VRUI.java

```diff
@@ -0,0 +1,224 @@
+		Rental r2 = new Rental(v2) ;
+		
+		james.addRental(r1) ;
+		james.addRental(r2) ;
+	}
+
+	public void listVideos() {
+		System.out.println("List of videos");
+		
+		for ( Video video: videos ) {
+			System.out.println("Price code: " + video.getPriceCode() +"\tTitle: " + video.getTitle()) ;
+		}
+		System.out.println("End of list");
+	}
+
+	public void listCustomers() {
+		System.out.println("List of customers");
+		for ( Customer customer: customers ) {
+			System.out.println("Name: " + customer.getName() +
+					"\tRentals: " + customer.getRentals().size()) ;
+			for ( Rental rental: customer.getRentals() ) {
+				System.out.print("\tTitle: " + rental.getVideo().getTitle() + " ") ;
+				System.out.print("\tPrice Code: " + rental.getVideo().getPriceCode()) ;
+			}
+		}
+		System.out.println("End of list");
+	}
+
+	public void getCustomerReport() {
+		System.out.println("Enter customer name: ") ;
+		String customerName = scanner.next() ;
+		
+		Customer foundCustomer = null ;
+		for ( Customer customer: customers ) {
+			if ( customer.getName().equals(customerName)) {
+				foundCustomer = customer ;
+				break ;
+			}
+		}
+
+		if ( foundCustomer == null ) {
+			System.out.println("No customer found") ;
+		} else {
+			String result = foundCustomer.getReport() ;
+			System.out.println(result);
+		}
+	}
+
+	public void rentVideo() {
+		System.out.println("Enter customer name: ") ;
+		String customerName = scanner.next() ;
+		
+		Customer foundCustomer = null ;
+		for ( Customer customer: customers ) {
+			if ( customer.getName().equals(customerName)) {
+				foundCustomer = customer ;
+				break ;
+			}
+		}
+
+		if ( foundCustomer == null ) return ;
+		
+		System.out.println("Enter video title to rent: ") ;
+		String videoTitle = scanner.next() ;
+
+		Video foundVideo = null ;
+		for ( Video video: videos ) {
+			if ( video.getTitle().equals(videoTitle) && video.isRented() == false ) {
+				foundVideo = video ;
+				break ;
+			}
+		}
+
+		if ( foundVideo == null ) return ;
+		
+		Rental rental = new Rental(foundVideo) ;
+		foundVideo.setRented(true);
+		
+		List<Rental> customerRentals = foundCustomer.getRentals() ;
+		customerRentals.add(rental);
+		foundCustomer.setRentals(customerRentals);		
+	}
+
+	public void register(String object) {
+		if ( object.equals("customer") ) {
+			System.out.println("Enter customer name: ") ;
+			String name = scanner.next();			
+			Customer customer = new Customer(name) ;
+			customers.add(customer) ;
+		}
+		else {
+			System.out.println("Enter video title to register: ") ;
+			String title = scanner.next() ;
+			
+			System.out.println("Enter video type( 1 for VHD, 2 for CD, 3 for DVD ):") ;
+			int videoType = scanner.nextInt();
+			
+			System.out.println("Enter price code( 1 for Regular, 2 for New Release ):") ;
+			int priceCode = scanner.nextInt();
+		
+			Date registeredDate = new Date();
+			Video video = new Video(title, videoType, priceCode, registeredDate) ;
+			videos.add(video) ;
+		}
+	}
+

```

Comment:
+ Please split register to Customer/Video reigester. It is 2 separate registration feature.